libopae/pluginmgr.c

```diff
@@ -135,12 +135,38 @@ STATIC char *find_cfg()
 	return NULL;
 }
 
+STATIC void *opae_plugin_mgr_find_plugin(const char *lib_path)
+{
+	char plugin_path[PATH_MAX];
+	const char *search_paths[] = {
+		"/usr/lib64/opae/",
```

Comment:
#define opae paths

Reply:
These appear only once within the given library.